scripts/automation/Whitesource_container_scan.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+source /root/cortx-re/solutions/kubernetes/functions.sh
+
+#Download WhiteSource Plugin 
+wget http://cortx-storage.colo.seagate.com/releases/cortx/security/whitesource/ws-k8s-agent.tar && tar -xvf ws-k8s-agent.tar >/dev/null 2>&1
+
+#user input
+read -p 'Please enter url: ' URL
+read -p 'Please enter apiKey: ' APIKEY
+read -p 'Please enter userKey: ' USERKEY
+read -p 'Please enter productName: ' PRODUCTNAME
+read -p 'Please enter docker registry: ' REGISTRY
+read -p 'Please enter mainPod details: ' MAINPOD
+read -p 'Please enter workerPod details: ' WORKERPOD
+read -p 'Please enter image pullSecret: ' PULLSECRET
+
+#Updating the configuration file
+sed -Ei "s,(url: ).*,\1$URL,g" /root/ws-k8s-agent/helm-chart/values.yaml; sed -Ei "s,(apiKey: ).*,\1$APIKEY,g" /root/ws-k8s-agent/helm-chart/values.yaml
```

Comment:
use variable for `values.yaml` path and use it wherever required

Reply:
while writing the groovy script for the parameters in jenkins job we will use it as mentioned below . The export variables are defined in the sed command  

string(name: 'WHITESOURCE_SERVER_URL', defaultValue: ' ', description: 'WhiteSource Server URL', trim: true)
string(name: 'API_KEY', defaultValue: ' ', description: 'apiKey', trim: true)
string(name: 'USER_KEY', defaultValue: ' ', description: 'userKey', trim: true)
string(name: 'PRODUCT_NAME', defaultValue: ' ', description: 'productName', trim: true)
string(name: 'DOCKER_REGISTRY', defaultValue: ' ', description: 'docker registry', trim: true)
string(name: 'MAIN_POD', defaultValue: ' ', description: 'mainPod details', trim: true)
string(name: 'WORKER_POD', defaultValue: ' ', description: 'workerPod details', trim: true)
string(name: 'PULL_SECRET', defaultValue: ' ', description: 'image pullSecretL', trim: true)

export WHITESOURCE_SERVER_URL=${WHITESOURCE_SERVER_URL}
export API_KEY=${API_KEY}
export USER_KEY=${USER_KEY}
export PRODUCT_NAME=${PRODUCT_NAME}
export DOCKER_REGISTRY=${DOCKER_REGISTRY}
export MAIN_POD=${MAIN_POD}
export WORKER_POD=${WORKER_POD}
export PULL_SECRET=${PULL_SECRET}